Frontend/android-native/app/src/driver/jatriSaathiDriver/assets/juspay/juspay_assets.json

```diff
@@ -236,6 +236,10 @@
       "ny_ic_chennai_metro" : true,
       "ny_ic_chennai_metro_map" : true,
       "ny_ic_chevron_arrow_right_white" : true,
+      "ny_ic_chennai_metro_banner" : true,
```

Comment:
@VD321 This assets are customer app related , why is it added here in driver

Reply:
Removed